rust-bins/wallet-notes/files/serialize_token_transfer_parameters-output.json

```diff
@@ -0,0 +1,3 @@
+{
+    parameter: "0000000000000049010001100100aa05dd88c26078e6315d051fa1e80cf67e9fcec66709cc2688e959864cef2ca500aa05dd88c26078e6315d051fa1e80cf67e9fcec66709cc2688e959864cef2ca50000"
```

Comment:
I've adjusted the input/output, but I don't think it is supposed to have the length prepended? :thinking: 
I guess that is the difference between the to_bytes method used.

Reply:
Yeah, the base16 should not be there. It is necessary that we use the contracts_common serial though since serialization of lengths inside the parameter, and integers, differs (it  is little endian).